src/index.js

```diff
@@ -1,149 +1,107 @@
-import deepEqual from 'deep-equal'
-
 // Constants
 
-export const UPDATE_PATH = '@@router/UPDATE_PATH'
+export const TRANSITION = '@@router/TRANSITION'
+export const UPDATE_LOCATION = '@@router/UPDATE_LOCATION'
+
 const SELECT_STATE = state => state.routing
 
-export function pushPath(path, state, { avoidRouterUpdate = false } = {}) {
-  return {
-    type: UPDATE_PATH,
-    payload: {
-      path: path,
-      state: state,
-      replace: false,
-      avoidRouterUpdate: !!avoidRouterUpdate
-    }
-  }
+function transition(method) {
+  return arg => ({
+    type: TRANSITION,
+    method, arg
+  })
 }
 
-export function replacePath(path, state, { avoidRouterUpdate = false } = {}) {
+export const push = transition('push')
```

Comment:
This could work:
```js
import { routeActions } from 'redux-simple-router'
routeActions.push('/foo')
```
It would also make using `bindActionCreators` easier because they're all under the same object namespace.